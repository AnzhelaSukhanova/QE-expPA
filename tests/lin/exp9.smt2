(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 4))
(declare-fun z () (_ BitVec 4))
(assert
 (exists ((x (_ BitVec 4))) (and
 (or (bvult x (bvadd (bvmul (bvshl #b0001 y) #b0011) (bvshl #b0001 z))) (= x (bvadd (bvmul (bvshl #b0001 y) #b0011) (bvshl #b0001 z))))
 (or (bvult (bvadd (bvshl #b0001 y) #b1010) x) (= (bvadd (bvshl #b0001 y) #b1010) x)))))
