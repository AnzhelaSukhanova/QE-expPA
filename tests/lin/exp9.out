(not (bvult (bvadd (bvshl #b0011 y) (bvshl #b0001 z)) (bvadd (bvshl #b0001 y) #b1010)))